.github/dependabot.yml

```diff
@@ -98,6 +98,29 @@ updates:
     ignore:
       - dependency-name: "*"
         update-types: ["version-update:semver-major"]
+  # Cloudsploit
+  - package-ecosystem: "docker"
+    directory: "/cloudsploit"
+    schedule:
+      interval: "weekly"
+    labels:
+      - "dependencies"
+      - "docker"
+      - "cloudsploit"
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-major"]
+  - package-ecosystem: "npm"
+    directory: "/cloudsploit"
+    schedule:
+      interval: "weekly"
+    labels:
+      - "dependencies"
+      - "npm"
+      - "cloudsploit"
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-major"]
```

Comment:
shouldn't it be `semver-minor` since you put version to `2` in `cloudsploit/config.yml` ?

Reply:
it's an ignore statement, so we'll only get minor & patches

Reply:
Oooh my bad. LGTM then